openwebui/patches/fix_large_tool_results.py

```
@@ -247,7 +247,7 @@ def apply_patch():
 
     print(f"  Large tool results patch applied! {changes}/3 modifications.")
     print(f"  Config: TOOL_RESULT_MAX_CHARS (default 50000), TOOL_RESULT_PREVIEW_CHARS (default 2000)")
-    print(f"  Upload: DOCKER_AI_UPLOAD_URL (optional, for Computer Use)")
+    print(f"  Upload: ORCHESTRATOR_URL (optional, for Computer Use)")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)